Bdd/features/environment.py

```
@@ -1,7 +1,42 @@
+
+
+def after_scenario(context, scenario):
+    # Clean up any long-lived interactive process
+    if hasattr(context, "interactive_process"):
+        process = context.interactive_process
+        if process.poll() is None:
+            process.stdin.write("quit\n")
+            process.stdin.flush()
+            try:
+                process.wait(timeout=5)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)